cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -65,6 +65,10 @@
       cronjob at:'*/1 * * * *', do:deploy_dir('bin', 'cron', 'update_dts')
     end
 
+    if node.chef_environment == 'staging' && node.name == 'staging-next'
```

Comment:
I think the `chef_environment` might be `test` for staging-next?  Not sure.

Reply:
This seems correct. Changing, and merging.